external-plugins/rehearse/plugin/handler/BUILD.bazel

```diff
@@ -12,7 +12,7 @@ go_library(
         "//vendor/k8s.io/test-infra/prow/config:go_default_library",
         "//vendor/k8s.io/test-infra/prow/git/v2:go_default_library",
         "//vendor/k8s.io/test-infra/prow/github:go_default_library",
-        "@io_k8s_test_infra//prow/pjutil:go_default_library",
+        "//vendor/k8s.io/test-infra/prow/pjutil:go_default_library",
```

Comment:
I think we should decide for the whole repo if it uses vendoring or not. Since most parts don't, I would probably remove it for this here too. What do others think?